website/pages/docs/autoscaling/plugins/apm.mdx

```diff
@@ -44,6 +44,50 @@ check {
 }
 ```
 
+## Datadog APM Plugin
+
+The [Datadog][datadog_homepage] APM allows using [time series][datadog_timeseries]
+data to make scaling decisions.
+
+### Agent Configuration Options
+
+```hcl
+apm "datadog" {
+  driver = "datadog"
+
+  config = {
+    dd_api_key = "<api key>"
+    dd_app_key = "<app key>"
+  }
+}
+```
+
+- `dd_api_key` `(string: "")` - The Datadog API key to use for authentication.
+- `dd_app_key` `(string: "")` - The Datadog APP key to use for authentication.
+
+The Datadog plugin can also read its configuration options via environment
+variables. The accepted keys are `DD_API_KEY` and `DD_APP_KEY`. The agent datadog
+configuration parameters take precedence over the environment variables.
+
+### Policy Configuration Options
+
+```hcl
+check {
+  source = "datadog"
+  query  = "FROM=2m;TO=0m;QUERY=avg:proxy.backend.response.time{proxy-service:web-app}"
+  ...
+}
+```
+
+The query consists of three sections, each separated using a `;` delimiter. More
+information on the arguments can be found on the [Datadog site][datadog_timeseries].
+
+- `FROM` - A time duration which indicates the start of the queried time period.
+
+- `TO` - A time duration which indicates the end of the queried time period.
```

Comment:
These two kind of confused me a bit. When making API requests directly they are absolute timestamps, but for the plugin query they are relative to the current time right?

I am not sure exactly how to explain this, but maybe it would be more clear to call it a time offset?

```suggestion
- `FROM` - A time offset which indicates the start of the queried time period.

- `TO` - A time offset which indicates the end of the queried time period.
```